src/bilder/images/odl_video_service/files/.env.tmpl

```diff
@@ -59,6 +59,7 @@ REDIS_MAX_CONNECTIONS="{{ keyOrDefault "ovs/redis_max_connections" "missing_valu
 STATUS_TOKEN="{{ keyOrDefault "ovs/environment" "missing_value_environment" }}"
 # In EC2 this will probably always be true
 USE_SHIBBOLETH="{{ keyOrDefault "ovs/use_shibboleth" "missing_value_use_shibboleth" }}"
+FEATURE_VIDEOJS_ANNOTATIONS="{{ keyOrDefault "ovs/feature_annotations" "missing_value_feature_annotations" }}"
```

Comment:
This might be a better one to set as a proper default since it's a binary option.